app/js/lib/speech-controller.js

```diff
@@ -102,14 +98,6 @@ export default class SpeechController extends EventDispatcher {
     this[p.speechSynthesis].speak(text);
   }
 
-  [p.initialiseSpeechRecognition]() {
-    return fetch(this[p.wakewordModelUrl])
-      .then((response) => response.json())
-      .then((model) => {
-        this[p.wakewordRecogniser].loadModel(model);
-      });
-  }
-
   [p.startListeningForWakeword]() {
     this.emit(EVENT_INTERFACE[0], { type: EVENT_INTERFACE[0] });
     this[p.idle] = true;
```

Comment:
I was thinking maybe we should keep this class and the dependency on jsSpeechRecognizer and rename so that we have 2 different speech-controller classes exposing the same API. Then this gives us the flexibility to do A/B testing or switch the keyword spotting backend easily.
What do you think?